docs/content/installation/upgrade_manual_openshift.md

```diff
@@ -34,4 +34,8 @@ When using the Red Hat® OpenShift® Container Platform, the CSI (Container Stor
 
 12. Check the **Overview** tab and that the **Controller Image Tab** and **Node Image Tag** are showing the most up-to-date version of the driver and the **Status** is _Running_.
```

Comment:
@matancarmeli7 is the same needed for host definer?

Reply:
where there is an "Overview" tab in openshift console?
In the hostDefiner there is a status field that one of his phases is `Running` and you can check his image if it's up-to-date, so yes this is something you can check also in the hostDefiner

Reply:
I've updated this whole section (as well as all openshift web sections - install/uninstall/upgrade).
please check and resole this thread, if possible. thanks